fileattrs/sysusers.attr

```diff
@@ -20,3 +20,23 @@
         ::continue::
     end
 }
+
+%__sysusers_dependencies() %{lua:
+    for line in io.lines(arg[1]) do
+        if line:sub(1, 1) == '#' then
+            goto continue
+        end
+        fields = {}
+        for w in line:gmatch("%S+") do
+            table.insert(fields, w)
+        end
+        if #fields >= 3 and fields[1] == 'm' then
+	    print(string.format('user(%s)\\ngroup(%s)', fields[2], fields[3]))
+        end
+        ::continue::
+    end
+}
+
+%__sysusers_recommends() %{?_use_weak_usergroup_deps:%{__sysusers_dependencies %1}}
+
+%__sysusers_requires() %{!?_use_weak_usergroup_deps:%{__sysusers_dependencies %1}}
```

Comment:
You'll need the %[] expression thing in these, like you had in the earlier version. %{?...} only tests whether the macro is defined, so this will use weak dependencies even if set to 0.